### PR TITLE
Install svn on ubuntu 24.04

### PR DESCRIPTION
**Pull request by johnbillion**
See https://github.com/actions/runner-images/issues/10636

> Removed from the Ubuntu 24.04 image due to maintenance reasons.

### CODE REVIEW

File: .github/workflows/reproduce.yml [hunk 1]
```diff
@@ -43,6 +43,13 @@ jobs:
           git clone --no-checkout git://develop.git.wordpress.org/ .
           git checkout "tags/${TAG}" -b "${TAG}"
 
+      - name: Install SVN
+        if: inputs.source == 'develop.svn.wordpress.org'
+        run: | #shell
+          sudo apt-get update
```
Comment (szepeviktor) [inline .github/workflows/reproduce.yml:49]: No need for `update`. It is automatic.

Comment (johnbillion) [reply .github/workflows/reproduce.yml:49]: Unfortunately [that's not true on GHA runners](https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/customizing-github-hosted-runners#installing-software-on-ubuntu-runners).

**Comment by johnbillion** [comment]
[5ded7f3](https://github.com/johnbillion/rave-wordpress/commit/5ded7f34b070a156e860821e484fcd31bf28adc9)

File: .github/workflows/reproduce.yml (hunk 1 above)
Comment (szepeviktor) [reply .github/workflows/reproduce.yml:49]: All right.
I think updating takes time and installation won't fail without it.

**Pull request closed without merging**